src/main/asciidoc/reference/elasticsearch-auditing.adoc

```diff
@@ -0,0 +1,196 @@
+[[elasticsearch.auditing]]
+= Auditing
+
+Spring Data provides sophisticated support to transparently keep track of who created or changed an entity and when the change happened. To benefit from that functionality, you have to equip your entity classes with auditing metadata that can be defined either using annotations or by implementing an interface. +
```

Comment:
A large part of this documentation is already available as [include](https://github.com/spring-projects/spring-data-mongodb/blame/master/src/main/asciidoc/index.adoc#L30) from [Spring Data Commons](https://github.com/spring-projects/spring-data-commons/blob/master/src/main/asciidoc/auditing.adoc).

Reply:
Didn't see that, I took a look at spring-data-mongodb, basic doc isn't referenced there ;-)

Reply:
Not sure: Do you have everything to bring the docs part into alignment with the other store modules or do I need to provide something?

Reply:
wanted to have a look at this this evening (I got a whole day of remote SAFe for Teams training today).
Can I build a local version with the included docs -`mvn -Pdistribute` build the docs locally, but I don't think that includes the common docs?
I'd like to have the common auditing docs and then a small section with the ES extras.

Reply:
Building the docs works indeed with `mvn install -Pdistribute`. Make sure to have Spring Data Commons checked out side-by-side with the ES project( e.g. `~/git/spring-data-commons`, `~/git/spring-data-elasticsearch`) as the docs include use relative paths outside of the actual Git repository.